ddtrace/span.py

```diff
@@ -83,7 +82,7 @@ def __init__(
         self.metrics = {}
 
         # timing
-        self.start = start or time.time()
+        self.start = time_ns() if start is None else (start * 1e9)
```

Comment:
I wonder if we can handle this gracefully, like `(start * 1e9) if not is_already_ns(start) else start`.

This comment adds a little unnecessary complexity here, just pondering out loud.

Reply:
Yeah, though there's no way to be *sure* it's already ns rather than guessing than a big number might need ns.

Another approach I'm just thinking about is that we could keep seconds but use `float` instead of `int` so if somebody passes a float it can pass nanoseconds and we can use nanoseconds by default too.